src/headset/headset.h

```diff
@@ -27,6 +28,7 @@ typedef enum {
   HEADSET_UNKNOWN,
   HEADSET_VIVE,
   HEADSET_RIFT,
+  HEADSET_OCULUS_MOBILE,
```

Comment:
Any way this can be more specific, like "OCULUS_GO"?  Assuming Oculus is exposing something like that...

Reply:
"Oculus Mobile" is the name Oculus gives to the platform which comprises the Samsung Gear, Oculus Go, and (I assume) Oculus Quest platforms.

What I am 98% sure I could do is use DRIVER_OCULUS_MOBILE for the driver, and then for the headset flag have HEADSET_GEARVR/'gearvr', HEADSET_OCULUS_GO/'oculusgo', and HEADSET_OCULUS_QUEST/'oculusquest' flags. (HEADSET_GEAR/'gear', HEADSET_GO/'go', and HEADSET_QUEST/'quest' would probably also be options). That work?

Reply:
yep, that is correct and sounds like the right approach for platform (driver) vs. headset. as you said, oculusvr is the platform name for all those mobile Oculus headsets.